docs/userguide/eng-image/dockerfile_best-practices.md

```diff
@@ -243,7 +243,7 @@ service (Apache, Rails, etc.), you would run something like
 recommended for any service-based image.
 
 In most other cases, `CMD` should be given an interactive shell (bash, python,
-perl, etc), for example, `CMD ["perl", "-de0"]`, `CMD ["python"]`, or
+perl, etc.), for example, `CMD ["perl", "-de0"]`, `CMD ["python"]`, or
```

Comment:
@YuPengZTE please address this comment (it got hidden somehow) and sign your commit using the -s option when you amend the commit. Let me know if you have more questions about this. Thank you so much for your submission.